### PR TITLE
Feature/catch error

### PR DESCRIPTION
**Pull request by cagallo**
#### What's this PR do?

- Adds function for display post message for the dropdown pantry
-  Adds function for the plus minus buttons on the rendered pantry

#### Where should the reviewer start?

- scripts.js
- index.html
- styles.css

#### How should this be manually tested?

- npm start
- go to localhost:8080
- add ingredient to pantry by dropdown pantry menu or by rendered pantry with the plus and minus buttons

#### Any background context you want to provide?

- N/A

#### What are the relevant tickets?

- N/A

### CODE REVIEW

File: src/apiCalls.js [hunk 1]
```diff
@@ -8,13 +8,14 @@ let recipeAPI = fetch("http://localhost:3001/api/v1/recipes")
...
 let post = async (info) => {
-  await fetch("http://localhost:3001/api/v1/users", {
+ let response = await fetch("http://localhost:3001/api/v1/users", {
     method: 'POST',
     body: JSON.stringify(info),
     headers: {
       'Content-Type': 'application/json'
     }
   })
+  return response;
```
Comment (anewb87) [inline src/apiCalls.js:18]: Needed a return for it to work. Yippee.

File: src/scripts.js [hunk 2]
```diff
@@ -85,32 +86,53 @@ addToPantryButton.addEventListener('click', addIngredientToPantry);
...
-// }
 
 async function postToPantry(info) {
+  try {
     let response = await post(info)
-    .catch(error => console.log(error))
+    let message = await response.json()
+    postMessage = message.message
+    console.log(postMessage)
+    // .then(response => {
```
Comment (anewb87) [inline src/scripts.js:96]: will get rid of these

File: src/scripts.js [hunk 3]
```diff
@@ -85,32 +86,53 @@ addToPantryButton.addEventListener('click', addIngredientToPantry);
...
-//   } catch (error) {
-//     console.log(error);
-//   }
-// }
 
 async function postToPantry(info) {
+  try {
     let response = await post(info)
-    .catch(error => console.log(error))
+    let message = await response.json()
```
Comment (anewb87) [inline src/scripts.js:93]: Pretty excited about how this is going.

File: src/scripts.js [hunk 4]
```diff
@@ -85,32 +86,53 @@ addToPantryButton.addEventListener('click', addIngredientToPantry);
...
+  }
+   catch(error) {
+    console.log(error)
+   }
+}
+
+function displayMessageDropdown() {
+  const dropdownMessage = document.querySelector('.dropdown-message');
+  dropdownMessage.innerText = `${postMessage}`;
+  setTimeout(() => {dropdownMessage.classList.add('fade-out')}, 2000);
```
Comment (anewb87) [inline src/scripts.js:110]: Look at that faaaaaade!

**Review by kammon10 — APPROVED** [review]
looks amazing!

File: src/styles.css [hunk 5]
```diff
@@ -91,10 +96,24 @@ h2 {
   color: #F2F5F2
 }
 
+.dropdown-message {
+  text-align: center;
+}
+
+.fade-out {
```
Comment (aliroemhildt) [inline src/styles.css:103]: excited about this new css trick!

Comment (anewb87) [inline src/styles.css:103]: This is super cool and super easy. For the double win!

**Review by aliroemhildt — APPROVED** [review]
looks good!

**Review by anewb87 — APPROVED** [review]
Awesome. Love how we're handing the messages. If we have time, we'll make them more user-friendly/specific.

**Pull request merged**